sample-compose/data/src/main/java/co/nimblehq/sample/compose/data/response/Response.kt

```diff
@@ -4,9 +4,16 @@ import co.nimblehq.sample.compose.domain.model.Model
 import com.squareup.moshi.Json
 
 data class Response(
-    @Json(name = "id") val id: Int?
+    @Json(name = "id")
+    val id: Int?,
+    // A sample to ensure @Json annotation work properly with a differ between field name and JSON key name
```

Comment:
How about this as the comment?
```suggestion
    // This field name is used to ensure @Json annotations work properly even if the field name and JSON key name are different.
```

Reply:
IMO, it's good enough. Thanks for your suggestion 🙏